app/javascript/app/components/search-categories-box/search-categories-box-styles.scss

```diff
@@ -1 +1,87 @@
 @import '~styles/ui-extends.scss';
+
+.container {
+  position: absolute;
+  width: 100%;
+}
+
+.inputWrapper {
+  border: 1px solid $mid-gray;
+  box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.09);
+}
+
+.inputCombo {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.checkbox {
+  margin-right: 10px;
+}
+
+.boxWrapper {
+  position: relative;
+  width: 100%;
+  bottom: 3px;
+  border: 1px solid $mid-gray;
+  box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.09);
+}
+
+.tabsContainer {
+  width: 100%;
+  display: flex;
+  justify-content: flex-start;
+  padding: 0 25px;
+  background-color: #f5f6f7;
+}
+
+.tab {
+  position: relative;
+  padding: 15px 0;
+  margin-right: 40px;
+
+  &::after {
+    content: '';
+    height: 0;
+    transition: all 0.2s;
+  }
+
+  &.active::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    background-color: $butterscotch;
+    height: 5px;
+    width: 100%;
+  }
+}
+
+.tabName {
+  @extend %bodyText;
+
+  color: $theme-color;
+  cursor: pointer;
+}
+
+.tabIndex {
+  font-size: 0.7rem;
+  color: $butterscotch;
+  position: absolute;
+  top: 10px;
+  right: -10px;
+  font-weight: 600;
+}
+
+.option {
+  padding-bottom: 25px;
+}
+
+.optionsContainer {
+  padding: 25px;
+
```

Comment:
Please add some background color white here. To avoid transparencies:

![image](https://user-images.githubusercontent.com/9701591/50084858-a5ff4d00-01f8-11e9-88ae-ee8c5fe6d395.png)